test_conformance/math_brute_force/main.cpp

```diff
@@ -774,7 +774,7 @@ test_definition test_list[] = {
     ADD_TEST( sinh ),
     ADD_TEST( sinpi ),
     ADD_TEST( sqrt ),
-    ADD_TEST( sqrt_cr ),
+    ADD_TEST_VERSION( sqrt_cr, Version(2, 0) ),
```

Comment:
It seems like this will exclude the `sqrt_cr` test completely?   But the ULP should still be tested for the non-relaxed case.

Sorry, I've not looked at how these kind of cases are generally handled.

Reply:
from what I can tell the `_cr` variants are the relaxed math tests, the regular `sqrt` test above remains unchanged

Reply:
I believe the `_cr` is for "correctly rounded": https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_API.html#CL_FP_CORRECTLY_ROUNDED_DIVIDE_SQRT